analysisstore/client/commands.py

```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```

Comment:
We should have a standardized nsls2 service config interface so
they're all the same and we have one place to change if we want to
switch something, like the host+port together or separate, and don't
have to keep rewriting the same thing.

-matt